src/main/java/uk/gov/hmcts/darts/courthouse/controller/CourthousesController.java

```diff
@@ -79,8 +79,8 @@ public ResponseEntity<Void> courthousesCourthouseIdPut(
     public ResponseEntity<List<ExtendedCourthouse>> courthousesGet(
 
     ) {
-        List<CourthouseEntity> courtHouseEntities = courthouseService.getAllCourthouses();
-        List<ExtendedCourthouse> responseEntities = mapper.mapFromListEntityToListExtendedCourthouse(courtHouseEntities);
+        List<CourthouseEntity> courthouseEntities = courthouseService.getAllCourthouses();
+        List<ExtendedCourthouse> responseEntities = courthouseService.getAllExtendedCourthouses(courthouseEntities);
```

Comment:
The method `getAllExtendedCourthouses` is named as a getter, but it has an arg. This is an unusual combination so I think this will cause some confusion to anyone casually reading this.

Suggest following a similar pattern to the other methods in this class and change it so something like `mapFromEntitiesToExtendedCourthouses`, which is a better description of what the method is doing.

Reply:
updated